src/molecules/side-menu.module.css

```
@@ -0,0 +1,92 @@
+.sideMenu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 22.5rem;
+    height: 100%;
+    background-color: var(--surface-color);
+    transform: translateX(-100%);
+    transition: transform 0.3s ease-in-out;
```

```suggestion
    transition: transform 0.15s ease-in-out;
```
ちょっとアニメーションが遅く感じるので, 半分くらいの長さでもよさそうです